styles/Page.module.scss

```diff
@@ -159,6 +159,12 @@
   }
 }
 
+@media only screen and (min-width: 675px) {
```

Comment:
are you doing this in this way because of mobile first?

Reply:
yeah